stdlib/array.gr

```diff
@@ -913,8 +913,7 @@ export let rotate = (n, arr) => {
   let rec gcd = (a, b) => {
     if (b == 0) {
       a
-    }
-    else{
+    } else {
```

Comment:
It's possible, just difficult because of our `EOL` tokens (for the same reason why we made the change to the infix operators). I can try to play around with it some more to see if I can get it to work.